pages/recording/upload.tsx

```diff
@@ -5,12 +5,14 @@ import { BlankViewportWrapper } from "ui/components/shared/Viewport";
 import UploadScreen from "ui/components/UploadScreen";
 import { useGetRecording, useGetRecordingId } from "ui/hooks/recordings";
 import { useGetUserSettings } from "ui/hooks/settings";
+import { useGetNonPendingWorkspaces } from "ui/hooks/workspaces";
 
 function UploadScreenWrapper({ onUpload }: { onUpload: () => void }) {
   const recordingId = useGetRecordingId();
   const { recording } = useGetRecording(recordingId);
-  // Make sure to get the user's settings before showing the upload screen.
+  // Make sure to get the user's settings and workspaces before showing the upload screen.
   const { userSettings, loading } = useGetUserSettings();
+  const { workspaces, loading: loading2 } = useGetNonPendingWorkspaces();
```

Comment:
I know you copied this over but while we're in here can we pick a more descriptive name for this stuff?
```suggestion
  const { userSettings, loading: userSettingsLoading } = useGetUserSettings();
  const { workspaces, loading: pendingWorkspacesLoading } = useGetNonPendingWorkspaces();
```